src/views/Change.vue

```diff
@@ -35,6 +35,13 @@
           :validate="getAppValidate"
         />
 
+        <DocumentsDelivery
+          class="mt-10"
+          sectionNumber="1."
```

Comment:
Can we use the CSS counter to automate section numbers?

Reply:
I looked at this, i'm not sure if it can be established on the parent and inherited into the sub components but can give it a try.

Reply:
Great. Don't spend a ton of time on it: it was like this when you got here 😁